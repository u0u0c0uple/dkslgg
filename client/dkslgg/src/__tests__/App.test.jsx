import { render } from "@testing-library/react";
import App from "../App";

const app = render(<App />);

describe("Example", () => {
  it("simple test", () => {
    expect(1 + 1).toBe(2);
  });
});

expect(app).toMatchSnapshot();