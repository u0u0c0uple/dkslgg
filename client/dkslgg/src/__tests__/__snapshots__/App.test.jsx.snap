// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <a
          href="https://vitejs.dev"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="Vite logo"
            class="logo"
            src="[object Object]"
          />
        </a>
        <a
          href="https://react.dev"
          rel="noreferrer"
          target="_blank"
        >
          <img
            alt="React logo"
            class="logo react"
            src="[object Object]"
          />
        </a>
      </div>
      <h1>
        Vite + React
      </h1>
      <div
        class="card"
      >
        <button>
          count is 
          0
        </button>
        <p>
          Edit 
          <code>
            src/App.jsx
          </code>
           and save to test HMR
        </p>
      </div>
      <p
        class="read-the-docs"
      >
        Click on the Vite and React logos to learn more
      </p>
      <div>
        0
      </div>
      <div
        class="sc-bcPKhP bmwHIs"
      >
        <div
          style="width: 100%; height: 100%;"
        />
      </div>
      <div
        class="sc-bcPKhP bmwHIs"
      >
        <div
          style="width: 100%; height: 100%;"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <a
        href="https://vitejs.dev"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="Vite logo"
          class="logo"
          src="[object Object]"
        />
      </a>
      <a
        href="https://react.dev"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="React logo"
          class="logo react"
          src="[object Object]"
        />
      </a>
    </div>
    <h1>
      Vite + React
    </h1>
    <div
      class="card"
    >
      <button>
        count is 
        0
      </button>
      <p>
        Edit 
        <code>
          src/App.jsx
        </code>
         and save to test HMR
      </p>
    </div>
    <p
      class="read-the-docs"
    >
      Click on the Vite and React logos to learn more
    </p>
    <div>
      0
    </div>
    <div
      class="sc-bcPKhP bmwHIs"
    >
      <div
        style="width: 100%; height: 100%;"
      />
    </div>
    <div
      class="sc-bcPKhP bmwHIs"
    >
      <div
        style="width: 100%; height: 100%;"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
